tests/integration/services/test_tei_integration.py

```
@@ -0,0 +1,341 @@
+        # Check reranker service
+        response = httpx.get("http://localhost:8081/health", timeout=5.0)
+        response.raise_for_status()
+    except httpx.RequestError:
+        pytest.skip("TEI services not available (run: docker compose up -d)")
+
+
+@pytest.mark.integration
+class TestRealTEIEmbedding:
+    """Test TEI embedding service with real service."""
+
+    def test_real_embedding_single_text(self, check_tei_services: None) -> None:
```

Ruff (0.14.0)

Unused method argument: `check_tei_services`

(ARG002)